app/routes/overview/components/Overview.js

```diff
@@ -13,59 +13,59 @@ import Feed from './Feed';
 import CompactEvents from './CompactEvents';
 import { EVENT_TYPE_TO_STRING, colorForEvent } from 'app/routes/events/utils';
 import Button from 'app/components/Button';
-import type { Event } from 'app/models';
 
 const TITLE_MAX_LENGTH = 50;
 const DESCRIPTION_MAX_LENGTH = 140;
 const IMAGE_HEIGHT = 192;
 
-function PrimaryItem({ event }: { event: ?Event }) {
-  if (!event) {
+function PrimaryItem({ item }: { item: ?Object }) {
+  if (!item) {
     return (
       <Flex column className={styles.primaryItem}>
-        <h2 className="u-ui-heading">Festet arrangement</h2>
+        <h2 className="u-ui-heading">Festet oppslag</h2>
         <Flex column className={styles.innerPrimaryItem}>
           <Image
             style={{ height: IMAGE_HEIGHT, display: 'block' }}
             className={styles.image}
             src={'https://i.redd.it/dz8mwvl4dgdy.jpg'}
           />
           <div className={styles.pinnedHeading}>
-            <h2 className={styles.itemTitle}>Ingen arrangementer</h2>
+            <h2 className={styles.itemTitle}>Ingen oppslag</h2>
           </div>
         </Flex>
       </Flex>
     );
   }
+  const base = item.eventType ? 'events' : 'articles';
```

Comment:
Maybe better to have a small helperfunction, that accepts different items, outside the render and returns the complete link?